Library/Homebrew/cmd/deps.rb

```diff
@@ -49,6 +49,8 @@ def self.deps_args
       switch "--installed",
              description: "List dependencies for formulae that are currently installed. If <formula> is " \
                           "specified, list only its dependencies that are currently installed."
+      switch "--skip-installed",
```

Comment:
Given your comments it feels like `--missing` is better here. Seems clearer/more explicit to say what is included rather than what is excluded.